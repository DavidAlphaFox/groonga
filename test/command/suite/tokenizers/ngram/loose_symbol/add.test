tokenize \
  'TokenNgram("loose_symbol", true)' \
  "090-1234-5678" \
  NormalizerAuto \
  --mode ADD
