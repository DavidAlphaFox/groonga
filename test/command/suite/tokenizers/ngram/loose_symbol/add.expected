tokenize   'TokenNgram("loose_symbol", true)'   "090-1234-5678"   NormalizerAuto   --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "090",
      "position": 0,
      "force_prefix": false
    },
    {
      "value": "-",
      "position": 1,
      "force_prefix": false
    },
    {
      "value": "1234",
      "position": 2,
      "force_prefix": false
    },
    {
      "value": "-",
      "position": 3,
      "force_prefix": false
    },
    {
      "value": "5678",
      "position": 4,
      "force_prefix": false
    },
    {
      "value": "ï¿°",
      "position": 5,
      "force_prefix": false
    },
    {
      "value": "09012345678",
      "position": 6,
      "force_prefix": false
    }
  ]
]
